swap/swap_test.go

```diff
@@ -52,6 +52,7 @@ import (
 	"github.com/ethereum/go-ethereum/rpc"
 	contractFactory "github.com/ethersphere/go-sw3/contracts-v0-2-0/simpleswapfactory"
 	"github.com/ethersphere/swarm/contracts/swap"
+	contract "github.com/ethersphere/swarm/contracts/swap"
```

Comment:
Duplicate (see line 56 below). Let's only keep one (`cswap` or `contract`)

Reply:
Whoops. Actually, it appears 3 times (see line 54 as well)...

Reply:
fixed